lib/template-loader.php

```diff
@@ -321,7 +321,7 @@ function gutenberg_find_template_post_and_parts( $template_type, $template_hiera
 
 	if ( $current_template_post ) {
 		$template_part_ids = array();
-		if ( is_admin() ) {
+		if ( is_admin() || defined( 'REST_REQUEST' ) ) {
```

Comment:
Will adding this affect anything other than the export theme request? (It seems like a fairly broad check)

Reply:
It will affect all REST requests. That's intended, and it's also needed in all the PRs that load the initial site editor state through REST.